app/views/articles/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2>Articles List</h2>
+
+<table>
+  <tr>
+    <th>Title</th>
+    <th>Text</th>
+  </tr>
+
+  <% @articles.each do |article| %>
```

Comment:
also, see test comments, below: this block is not tested, and represents the bulk of the code